### ISSUE TITLE
Traffic Layer

### ISSUE THREAD

**Issue by Wgil** [mapgears/ol3-google-maps#42]
Hello, I have an application that uses OpenLayers 3 and for what Ive been searching, Google Maps is not officially supported by ol3.

is it possible to add the google traffic layer to ol3 with this library?

Someone has tried it?

Thank you in advance.

**Comment by adube** [comment]
Hi @Wgil.  Thanks for your interest in OL3-Google-Maps !

It's currently not possible, but should be fairly easy to accomplish.  OL3GM creates the `google.maps.Map` object iself and currently has not mean to get it.  We could add a getter to get that map, then it should be easy to add the traffic layer thereafter.

It hasn't been tested yet.  I'd be willing to work on this, but we're currently facing issue with the compilation of the library and the latest version of openlayers (documented in #33).  We're currently waiting for the fixes to be made in ol3, but I expected them to be made sooner.

Bottom line: if you can wait a couple of days/weeks for this feature, I'd be willing to work on it as it should be easy to do.  I'd add an example that demonstrate this.

**Comment by Wgil** [comment]
Hello @adube 

Thanks for answering, I will be attentive with this. Tell me if you need some help.

**Comment by PabloDesiderioFlores** [comment]
Hi, I have the same problem, I need to use google as a layer and then use the traffic layer from google too.

**Comment by adube** [comment]
If you're okay with waiting, I'm still willing to work on this but my current schedule is booked for a while.  If any of you is interested in working on a Pull Request, I'd be happy to review.  Thanks!

**Comment by adube** [comment]
I added an example that demonstrate the traffic layer in #53.  See it in action: http://dev5.mapgears.com/ol3-google-maps/examples/traffic.html

Until a release is made, you'll need to build your version of ol3gm or pick the one from dev5.

**Comment by Wgil** [comment]
Thank you very much!

**Comment by knandhucse** [comment]
Hello, Any update on using the traffic layer inside the open layer map. Please help me on how to achieve that.

**Comment by adube** [comment]
If you're using ol3-google-maps v0.20.0 or lower, see how it's done in this example:
 * http://mapgears.github.io/ol3-google-maps/examples/dist/examples/traffic.html

If you're using v1.0.0-beta.4, see the source code of the same example (updated to use modules):
 * https://github.com/mapgears/ol3-google-maps/blob/master/examples/traffic.js

HTH
